From ed86f00a30b929612e48623d6cf11270681cb910 Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Mon, 30 Sep 2013 10:16:59 +0200
Subject: [PATCH] nach

---
 default.prop | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/default.prop b/default.prop
index d1906dc..8f75cd0 100644
--- a/default.prop
+++ b/default.prop
@@ -1,8 +1,8 @@
 #
 # ADDITIONAL_DEFAULT_PROPERTIES
 #
-ro.adb.secure=1
-ro.secure=1
+ro.adb.secure=0
+ro.secure=0
 ro.allow.mock.location=0
 ro.debuggable=1
 ro.zygote=zygote32
-- 
1.8.1.2

