From 25db9e7efbb763cc6138549fe75d8ec71bc4f247 Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Tue, 8 Mar 2016 08:03:45 +0100
Subject: [PATCH] 3

---
 file_contexts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/file_contexts b/file_contexts
index 78d9b1c..de4d89d 100644
--- a/file_contexts
+++ b/file_contexts
@@ -870,3 +870,6 @@
 
 # bash
 /system/xbin/bash                               u:object_r:shell_exec:s0
+
+# Boeffla specific selinux contexts
+/sbin/busybox			u:object_r:toolbox_exec:s0
--
1.9.1

