From 3c334ea43259b9cbfdc10102655fc7d952227a3f Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Mon, 14 Dec 2015 08:22:47 +0100
Subject: [PATCH] 2

---
 init.rc | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/init.rc b/init.rc
index 5c6b606..97c42df 100755
--- a/init.rc
+++ b/init.rc
@@ -769,3 +769,10 @@ service logcatd /system/bin/logcat -b all -v threadtime -v usec -v printable -D
     user logd
     group log
     writepid /dev/cpuset/system-background/tasks
+    
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init.sh
+    class main
+    user root
+    group root
+    oneshot    
-- 
1.9.1

